### ISSUE TITLE
The blueprint is not extending from AbstractAuditingEntity 

### ISSUE THREAD

**Issue by mbrandl87** [jhipster/generator-jhipster-entity-audit#210]
Hi There,

according to the readme, every entity should be extending AbstractAuditingEntity  after applying the blueprint. 
I did that by calling jhipster --blueprints entity-audit and got asked the audit related questions.
Afterwards no entity except user extens AbstractAuditingEntity  though.

I used jhipster 8.0.0.beta-3 together with generator-jhipster-entity-audit#main.

Best and thx 
Martin

**Comment by mshima** [comment]
Main has changed to jhipster main which has breaking changes. It won’t work with beta.3.
You can
- use `jhipster-entity-audit` command.
- use jhipster main.
- use entity-audit 5.0.0-beta.1

**Comment by mbrandl87** [comment]
Hi,

thanks for such a quick reply. But I still cannot get this to work.
Ah, sorry if you get my reply twice now. I answered by email yesterday but it isn't appearing here so far :)

First I did an npm i -g 
generator-jhipster-entity-audit@5.0.0-beta.1
 to be able to use the jhipster-entity-audit command.

Then I replaced it globally with latest stable 4.0 and tried my luck using a jhipster 7.8.1 project.

It doesn't matter if I chose custom or javers, in neither case any of the entities extends the AbstractAuditingEntity after the generator finishes.

I selected to update all entities and gave an a to the yeoman conflict prompt.

What am I missing here? :)

Best and thx 
Martin
